ocs_ci/ocs/resources/pod.py

```diff
@@ -177,6 +177,19 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
             return [item for item in out if item]
         return out
 
+    def get_mount_path(self):
+        """
+        Get the pod volume mount path
+
+        Returns:
+            str: The mount path of the volume on the pod (e.g. /var/lib/www/html/)
+        """
+        return (
+            self.pod_data.get(
+                'spec'
+            ).get('containers')[0].get('volumeMounts')[0].get('mountPath')
```

Comment:
I would like to get this path for later usage in the new IO in BG fixture. Any suggestion of doing it differently?

Reply:
The problem here is that this is a method of general purpose class, but getting mount path of 1st volume of 1st container in a pod is quite specific and makes sense only in some particular cases, such as for the container you run FIO workload in. Moreover the code is quite trivial, and not worth the effort wrt the issues mentioned above and additional layer of abstraction we would create this way.

Reply:
That said, maybe the assumption so far is that we will always use containers and volumes in a way so that this will work. With that in mind, you can keep it here for now (but later we could run into problems if these unstated assumptions changes).

But in general and long term perspective, it looks like we are building bit pointless over complicated wrapper objects for everything, which I'm not really happy about.

Reply:
In any case, also without this method, the existing usage of it, in pod class, run.io(), fetches the 1st vol in the 1st container.
I agree this is not flexible and could cause issues in the future but I think we'll be able to change this code easily

Reply:
Ok, adding a todo comment about this may be enough for this PR.

Reply:
done